### ISSUE TITLE
SendTo sync with main thread

### ISSUE THREAD

**Issue by dungeon2567** [MidLevel/Ruffles#23]
**Is your feature request related to a problem? Please describe.**
Can SendTo be triggered by the main thread? i have a game loop on my game server and want to send all messages at the end of every step
**Describe the solution you'd like**
Main Thread trigger Event at end of loop, SendTo Thread "WaitsFor" Event to run logic

**Describe alternatives you've considered**

**Additional context**

**Comment by TwoTenPvP** [comment]
All API's in Ruffles are 100% thread safe and many are threaded under the hood already.

**Comment by dungeon2567** [comment]
Yes, they are thread safe but i want to send all messages exactly on the end of current server tick to optimize latency (my server is already 20 fps, 50 ms per frame)

**Comment by dungeon2567** [comment]
Is there anything wrong with this? because even even though both threads run at 20 ticks per second they are not in sync (i write all  data that need to be sent this frame before the frame end and i can flush it now)
